components/director/cmd/director/main.go

```diff
@@ -281,3 +303,22 @@ func getRuntimeMappingHanderFunc(transact persistence.Transactioner, cachePeriod
 		runtimeSvc,
 		tenantSvc).ServeHTTP, nil
 }
+
+func createMetricsServer(metricsAddress string) *http.Server {
+	metricsHandler := http.NewServeMux()
+	metricsHandler.Handle("/metrics", promhttp.Handler())
+
+	return &http.Server{Addr: metricsAddress, Handler: metricsHandler}
+}
+
+func runServer(srv *http.Server, name string) {
```

Comment:
Maybe this function can return the shutdown function for the given server and name? Right now you need to call `runServer` and `shutdownServer` with similar parameters.

Reply:
Very good idea, I will do it